### PR TITLE
Fix TorchVision tests caused by file renaming

### PR DESCRIPTION
**Pull request by ejguan**
This PR is required since https://github.com/pytorch/vision/pull/6587 has been landed and shipped to TorchVision nightly.

### CODE REVIEW

File: .github/workflows/domain_ci.yml [hunk 1]
```diff
@@ -56,7 +56,7 @@ jobs:
           path: vision
 
       - name: Run torchvision builtin datasets tests
-        run: pytest --no-header -v vision/test/test_prototype_builtin_datasets.py
+        run: pytest --no-header -v vision/test/test_prototype_datasets*.py
```
Comment (pmeier) [inline .github/workflows/domain_ci.yml:59]: You probably want

```suggestion
        run: pytest --no-header -v vision/test/test_prototype_datasets_builtin.py
```

Otherwise you will also run our utility tests and the like that are unrelated to `torchdata`.

**Review by pmeier — APPROVED** [review]
Thanks, I totally forgot!

File: .github/workflows/domain_ci.yml (hunk 1 above)
Comment (pmeier) [reply .github/workflows/domain_ci.yml:59]: Or if you actually want to run all tests, you need to use 

```suggestion
        shell: bash
        run: pytest --no-header -v vision/test/test_prototype_datasets*.py
```

to be able to use the `*.py` globbing

Comment (ejguan) [reply .github/workflows/domain_ci.yml:59]: Sounds great. Thank you for the suggestion.

**Review by NivekT — APPROVED** [review]
